CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # CHANGE LOG
+## [0.9.2]
```

Comment:
So we won't have released 9.1 (or 9.0) before this is in, should we be adding to the change log instead of skipping versions?

Reply:
oh yeah, that's just an oversight on my part then, I'll switch this back to 9.1